### ISSUE TITLE
output files

### ISSUE THREAD

**Issue by peterthorpe5** [mortazavilab/lapa#9]
Dear Lapa,

I have run your tool, currently as a test. I was wondering if you could add some documentation/ pass on some wisdom  on what output files to expect?

I have bw and bed files for all my conditions. I have polyA_cluster.bed (what do the columns stand for? What does "None@None" in the last columns mean?  . 

I have no other files ... ...  I was wondering if the program terminated early, however, there is no error or warning in any out files via slurm .. (RAM limit was not exceeded)

can you please advise?

Kind regards,

Pete

**Comment by MuhammedHasan** [comment]
Thanks for your interest in LAPA. I am working on the new release of LAPA 0.0.3 will be out next week when I add more documentation.

**Comment by peterthorpe5** [comment]
Hi Muhammed, 

I was wondering if there was some more documentation on the output file (even column headers please)?  I have results and I am keen to understand them. :)

Is this on a dev branch?

regards, Pete

**Comment by MuhammedHasan** [comment]
Hi Peter, 

Thanks for your interest and patience. I released version 0.0.3 of LAPA, which is the stable version of the package. Also, documentation is added for output format: https://lapa.readthedocs.io/en/tss_clustering/output.html

I highly recommend you rerun the new version where I implemented new features to improve cluster calling especially filtering for replication. 

Thanks for your interest.
